vpr/src/pack/re_cluster.cpp

```diff
@@ -28,44 +41,137 @@ bool move_atom_to_new_cluster(const AtomBlockId& atom_id,
 
     if (helper_ctx.num_used_type_instances[block_type] == num_instances) {
         VTR_LOG("The utilization of block_type %s is 100%. No space for new clusters\n", block_type->name);
-        VTR_LOG("Atom %d move aborted\n", atom_id);
+        VTR_LOG("Atom %d move aborted\n", molecule->atom_block_ids[molecule->root]);
```

Comment:
I suspect this is going to have to be guarded by VERBOSE with a VTR_LOGV and a verbose level. This could produce a lot of output once you start iteratively improving clusters and the chip is full.